Packs/SilentPush/Integrations/SilentPush/SilentPush.yml

```diff
@@ -323,6 +454,35 @@ script:
     - contextPath: SilentPush.DomainASNs.asns
       description: Dictionary of Autonomous System Numbers (ASNs) associated with the domain.
       type: Unknown
+  - deprecated: false
+    description: 'This command runs the threat check on the specified '
+    name: silentpush-get-data-exports
+    arguments:
+    - name: feed_url
+      isArray: false
+      description: The URL from which to export the feed data.
+      required: true
+      secret: false
+      default: false
+    outputs:
+    - contextPath: SilentPush.GetDataExports.EntryID
+      description: The EntryID of the report file.
+      type: Unknown
+    - contextPath: SilentPush.GetDataExports.Extension
+      description: The extension of the report file.
+      type: String
+    - contextPath: SilentPush.GetDataExports.Name
+      description: The name of the report file.
+      type: String
+    - contextPath: SilentPush.GetDataExports.Info
+      description: The info of the report file.
+      type: String
+    - contextPath: SilentPush.GetDataExports.Size
+      description: The size of the report file.
+      type: Number
+    - contextPath: SilentPush.GetDataExports.Type
+      description: The type of the report file.
+      type: String
   - deprecated: false
```

Comment:
why didn't you return it as an infofile? you're returning a fileResult in the code